Makefile

```diff
@@ -1,34 +1,14 @@
-.PHONY: all clean install build
-PREFIX ?= /usr/local
+all:
+	jbuilder build @install -j 3
```

Comment:
should this be `jbuilder build --dev` (for stricter warnings) and then it autofigures out the right value of `-j` (i have 16 cores on my desktop)